src/engine.ts

```diff
@@ -451,6 +451,15 @@ export default class Engine {
       data: fromCommand
     });
   }
+
+  buildMinePhase(player: PlayerEnum, benefit: string){
```

Comment:
Can be removed

Reply:
I moved your code here, to be consistent with the other phases. Check if yo like it